### PR TITLE
Converted item weights to user's preferred unit

### PR DESCRIPTION
**Pull request by MuhammadHassan03**

### CODE REVIEW

File: packages/app/modules/pack/components/PackTable/TableContainer.tsx [hunk 1]
```diff
@@ -61,6 +61,7 @@ export const TableContainer = ({
   if (isLoading) return <RSkeleton style={{}} />;
   if (error) return <ErrorMessage message={String(error)} />;
 
+
```
Comment (taronaleksanian) [inline packages/app/modules/pack/components/PackTable/TableContainer.tsx:64]: Please remove this

File: packages/ui/src/Bento/elements/tables/Basic.tsx [hunk 2]
```diff
@@ -217,6 +253,8 @@ export function BasicTable({
 
   const { sm } = useMedia();
 
+  console.log('groupedData', groupedData)
```
Comment (taronaleksanian) [inline packages/ui/src/Bento/elements/tables/Basic.tsx:256]: please remove console.logs

File: packages/app/modules/pack/components/PackTable/TableContainer.tsx (hunk 1 above)
Comment (MuhammadHassan03) [reply packages/app/modules/pack/components/PackTable/TableContainer.tsx:64]: My bad, will remove console.logs soon

File: packages/ui/src/Bento/elements/tables/Basic.tsx [hunk 3]
```diff
@@ -58,14 +60,18 @@ export function BasicTable({
   refetch,
   setRefetch,
 }: BasicTableProps) {
+  const { user } = useProfile(null);
+  console.log('user', user.preferredWeight)
```
Comment (taronaleksanian) [inline packages/ui/src/Bento/elements/tables/Basic.tsx:64]: please remove console.logs

File: packages/ui/src/Bento/elements/tables/Basic.tsx [hunk 4]
```diff
@@ -84,6 +90,10 @@ export function BasicTable({
       openModal('delete', item.id);
     };
 
+
+
+
```
[the remark this answers is not in the document]
Comment (MuhammadHassan03) [reply packages/ui/src/Bento/elements/tables/Basic.tsx:95]: Spacing?

File: packages/ui/src/Bento/elements/tables/Basic.tsx [hunk 5]
```diff
@@ -58,14 +60,18 @@ export function BasicTable({
   refetch,
   setRefetch,
 }: BasicTableProps) {
+  const { user } = useProfile(null);
```
Comment (taronaleksanian) [inline packages/ui/src/Bento/elements/tables/Basic.tsx:63]: we have useAuthUser hook for getting the user. please use it here

File: packages/ui/src/Bento/elements/tables/Basic.tsx (hunk 3 above)
Comment (MuhammadHassan03) [reply packages/ui/src/Bento/elements/tables/Basic.tsx:64]: Sure

File: packages/ui/src/Bento/elements/tables/Basic.tsx (hunk 5 above)
Comment (MuhammadHassan03) [reply packages/ui/src/Bento/elements/tables/Basic.tsx:63]: Sure

File: packages/ui/src/Bento/elements/tables/Basic.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (taronaleksanian) [reply packages/ui/src/Bento/elements/tables/Basic.tsx:95]: yeah

**Review by taronaleksanian — APPROVED** [review]

**Pull request merged**